superset/models/core.py

```diff
@@ -189,6 +189,12 @@ def allows_cost_estimate(self) -> bool:
             and cost_estimate_enabled
         )
 
+    @property
+    def allows_virtual_table_explore(self) -> bool:
+        extra = self.get_extra()
+
+        return bool(extra.get("virtual_table_explore_enabled", True))
```

Comment:
nit: I'd prefix this with `is` since it's a boolean

Reply:
This is probably for keeping consistency with other properties:

```
            "allow_multi_schema_metadata_fetch": self.allow_multi_schema_metadata_fetch,
            "allows_subquery": self.allows_subquery,
            "allows_cost_estimate": self.allows_cost_estimate,
            "allows_virtual_table_explore": self.allows_virtual_table_explore,
```

I'd say the more bothering thing is "allow" vs "allows"...